scripts/kms/endpoints/key.sh

```diff
@@ -48,6 +48,8 @@ key() {
     auth_arg=()
     if [[ "$auth" == "member_cert" ]]; then
         auth_arg=(--cert $KMS_MEMBER_CERT_PATH --key $KMS_MEMBER_PRIVK_PATH)
+    elif [[ "$auth" == "user_cert" ]]; then
+        auth_arg=(--cert $KMS_USER_CERT_PATH --key $KMS_USER_PRIVK_PATH)
     elif [[ "$auth" == "jwt" ]]; then
         auth_arg=(-H "Authorization: Bearer $(curl -X POST $JWT_ISSUER | jq -r '.access_token')")
```

Comment:
might be good to show an error if the wrong auth value is passed. Will be hard to debug without an error

Reply:
What do you mean by wrong? The `/key` endpoint allows all three types of authentication, but if the user provides a value that the endpoint rejects shouldn't it already give you a suitable error?